main.go

```diff
@@ -151,7 +156,7 @@ func generatorLoop(c *cli.Context, exec *executor.Executor) {
 		if err != nil {
 			panic(err)
 		}
-		if len(infos) > minTests {
+		if len(infos) < minTests {
```

Comment:
I think this is wrong.
The idea behind this whole loop was to start the generation and the execution and stop the generation when maxTests was hit. 
Then the executor has some time to execute all tests and delete the non failing tests.
If more than `minTests` fail, the executor should stop, else the generator should start again